Assets/Scripts/Models/Tile.cs

```diff
@@ -112,7 +112,7 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovementModifier { get; set; }
+    public float MovementModifier;
```

Comment:
It was changed in a previous PR to be public so that it can be changed by LUA scripts. The accessor verbiage was left in, which was removed to make it look cleaner, both of which I believe are the correct decisions.